src/node_messaging.cc

```diff
@@ -293,6 +306,19 @@ class SerializerDelegate : public ValueSerializer::Delegate {
           BaseObjectPtr<BaseObject> { Unwrap<BaseObject>(object) });
     }
 
+    // Convert process.env to a regular object.
+    auto env_proxy_ctor_template = env_->env_proxy_ctor_template();
+    if (!env_proxy_ctor_template.IsEmpty() &&
+        env_proxy_ctor_template->HasInstance(object)) {
+      HandleScope scope(isolate);
+      // TODO(bnoordhuis) Prototype-less object in case process.env contains
+      // a "__proto__" key?
+      Local<Object> normal_object = Object::New(isolate);
```

Comment:
This feels like a requirement no?

```suggestion
      // TODO(bnoordhuis) Prototype-less object in case process.env contains
      // a "__proto__" key?
      Local<Object> normal_object = Object::New(isolate, v8::Null(isolate));
```

Reply:
I'm a little undecided on this.

The values are strings so there's no real risk of prototype pollution, and _not_ having a prototype means no e.g. `toString()`. Might be confusing because `process.env` has it.

Reply:
> Might be confusing because `process.env` has it.

Well, this falls under "Today I Learned"... it sure seems wrong because none of the methods' presences can be relied upon, but it's a fair point, but I get then that this makes sense in this PR for consistency.